doc/cli.rst

```diff
@@ -283,13 +283,13 @@ Example
 
 Example::
 
-    $ python3 -m perf timeit 1+1
+    $ pybench timeit 1+1
     .........................
     Median +- std dev: 11.7 ns +- 0.1 ns
 
 Use ``-v`` to enable the verbose mode::
 
-    $ python3 -m perf timeit -v 1+1
+    $ pybench timeit -v 1+1
```

Comment:
I may add a parameter to specific the Python binary, but until a parameter is added, I prefer to write "python3 -m perf timeit ..." just to be more explicit about the tested Python version.